databroker/src/grpc/kuksa_val_v2/val.rs

```diff
@@ -146,30 +190,145 @@ impl proto::val_server::Val for broker::DataBroker {
+            match opt_signal_id {
+                Some(signal_id) => {
+                    match signal_id.signal {
+                        Some(proto::signal_id::Signal::Path(vss_path)) => {
+                            if let Some(id) = broker.get_id_by_path(&vss_path).await {
+                                let result = broker.actuate(&id, &DataValue::from(value)).await;
+                                match result {
+                                    Ok(_) => return Ok(tonic::Response::new(ActuateResponse {})),
+                                    Err(error) => {
+                                        return Err(convert_actuation_error_to_status(
+                                            error.0, error.1,
+                                        ))
+                                    }
+                                };
+                            }
+                            return Err(tonic::Status::not_found(format!(
+                                "Invalid vss_path provided {}",
+                                vss_path
+                            )));
+                        }
+                        Some(proto::signal_id::Signal::Id(vss_id)) => {
+                            let result = broker.actuate(&vss_id, &DataValue::from(value)).await;
+                            match result {
+                                Ok(_) => return Ok(tonic::Response::new(ActuateResponse {})),
+                                Err(error) => {
+                                    return Err(convert_actuation_error_to_status(error.0, error.1))
+                                }
+                            };
+                        }
+                        None => {
+                            return Err(tonic::Status::invalid_argument(
+                                "Signal needs to provide Path or Id",
+                            ))
+                        }
+                    };
+                }
+                None => return Err(tonic::Status::invalid_argument("No Signal_Id provided")),
+            };
+        };
+        return Err(tonic::Status::invalid_argument(
+            "Invalid Actuator Request provided",
+        ));
     }
 
+    // Actuate simultaneously multiple actuators.
+    // If any error occurs, the entire operation will be aborted
+    // and no single actuator value will be forwarded to the provider.
+    //
+    // Returns (GRPC error code):
+    //   NOT_FOUND if any of the actuators are non-existant.
+    //   PERMISSION_DENIED if access is denied for any of the actuators.
+    //   UNAVAILABLE if there is no provider currently providing an actuator
+    //   INVALID_ARGUMENT
+    //       - if the data type used in the request does not match
+    //            the data type of the addressed signal
+    //       - if the requested value is not accepted,
+    //            e.g. if sending an unsupported enum value
     async fn batch_actuate(
```

Comment:
If we would respond with some `BatchActuateResponse` in the future I think we need to split this as well into `batch_actuate_id` and `batch_actuate`. Otherwise the client could mix paths and ids and when getting back a response he could be confused. Or am I wrong @rafaeling @argerus ?